client/src/components/BizCard/BizCard.jsx

```diff
@@ -70,6 +73,15 @@ export default function BizCard({ props }) {
     "pk_test_51HU0G2CjwFEQ1pgcvOchnwo0Gsb2seN5a3xGz8Q2iCvlVUjHkSCV7UZHy3NfeobxNNMeGwmiosi3UBxjbKcSjGZ000hENfQW0F"
   );
 
+  
+  //get difference between hours to send to stripe to get price
+  function hoursHandler() {
```

Comment:
can you use this to update the "Booking" section so that the total price is a reflection of hourly cost + total hours....something like: "Total Price for {numberOfHours} hours: ¥2500"

Reply:
fixed